src/common/xio_session.c

```diff
@@ -358,6 +358,8 @@ void xio_session_notify_connection_established(
 #ifdef XIO_THREAD_SAFE_DEBUG
 		xio_ctx_debug_thread_unlock(connection->ctx);
 #endif
+                xio_ctx_del_delayed_work(connection->ctx,
```

Comment:
Please fix the spacing (tabs, etc...) according to coding conventions throughout your changes in this file.